cycledash/static/js/runs/components/RunsPage.js

```diff
@@ -386,6 +386,36 @@ var RunLabels = React.createClass({
   }
 });
 
+var RunBAMs = React.createClass({
+  propTypes: {
+    run: React.PropTypes.object.isRequired
+  },
+  render: function() {
+    var run = this.props.run;
+
+    var labelSpecs = [
+      {path: ['tumor_bam' , 'name'], usePathValue: true, title: 'Tumor BAM', cssClass: 'linked-bam'},
```

Comment:
Good call. I removed `usePathValue` from both the status and linked bam components.